docs/source/science/OPERA/OPERA_Surface_Displacement.ipynb

```diff
@@ -0,0 +1,5744 @@
+{
```

Comment:
<p>These are also already installed in the MAAP Pangeo image. see above comment.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>